src/widget/widget.cpp

```diff
@@ -84,8 +84,10 @@ Widget::Widget(QWidget *parent)
       eventFlag(false),
       eventIcon(false)
 {   
+    installEventFilter(this);
     translator = new QTranslator;
     setTranslation();
+    WasMaximized = false;
```

Comment:
This is C++11, you can initialize this from the declaration. Also the first w should be lower case.